shinysdr/i/webstatic/client/values.js

```diff
@@ -346,7 +346,20 @@ define([
   function isImplementing(object, interfaceName) {
     return !!object['_implements_' + interfaceName];
   }
-  exports.getInterfaces = getInterfaces;
+  exports.isImplementing = isImplementing;
+
+  function findImplementers(array, interfaceName) {
+    const out = [];
+    for (const key in array) {
+      // TODO: By using .get() here, we assume that elements will not suddenly start or stop implementing an interface.
```

Comment:
You can take a dirty callback for `.depend()` as a parameter to fix this, or write it as a `DerivedCell` factory instead of a function.